src/api/props.js

```diff
@@ -1,15 +1,16 @@
 import { renderer as $renderer } from '../util/symbols';
 import assign from '../util/assign';
 
-function get (elem) {
+function get(elem) {
   const props = {};
-  for (let key in elem.constructor.props) {
+  Object.keys(elem.constructor.props).forEach((key) => {
```

Comment:
We use a `for..in` loop at https://github.com/skatejs/skatejs/pull/701/files#diff-d3ff8469a13e890ec2a5c73b90df75abR185 - can we use it here as well?